core/arch/arm/kernel/thread.c

```diff
@@ -593,6 +594,7 @@ void __thread_std_smc_entry(struct thread_smc_args *args)
 		tee_fs_rpc_cache_clear(&thr->tsd);
 		if (!thread_prealloc_rpc_cache) {
 			thread_rpc_free_arg(thr->rpc_carg);
+			mobj_free(thr->rpc_mobj);
```

Comment:
suggested `thr->rpc_mobj = NULL`, but i could not insure this is really required.